src/eslint/rules/__tests__/exaustive-inject.test.js

```diff
@@ -0,0 +1,166 @@
+import { RuleTester } from 'eslint';
+import rule from '../exhaustive-inject';
+
+RuleTester.setDefaultConfig({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+    ecmaFeatures: { jsx: true },
+  },
+});
+
+var ruleTester = new RuleTester();
+ruleTester.run('exhaustive-inject', rule, {
+  valid: [
+    // it should ignore built-in react components
+    `
+      import { Fragment } from 'react';
+      import { di } from 'react-magnetic-di';
+      import { useQuery } from 'react-apollo';
+
+      function MyComponent() {
+        di(useQuery);
```

Comment:
should this inject `Fragment`?

Reply:
No, this one is fine, we ignore `Fragment` too